src/test/java/com/spotify/folsom/transcoder/SerializableObjectTranscoderTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2015 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.spotify.folsom.transcoder;
+
+import org.junit.Test;
+
+import java.io.Serializable;
+
+import static org.junit.Assert.assertEquals;
+
+public class SerializableObjectTranscoderTest {
+
+  private static final class SerializableTestObject implements Serializable {
+    private String value1 = "hello";
```

Comment:
Good catch! Though I guess it doesn't matter since we don't need a consistent serial id for a very short lived test class/object.